src/components/layout/Menu/MenuStyled.js

```diff
@@ -1,60 +1,93 @@
-import MuiDrawer from "@mui/material/Drawer";
-import { styled } from "@mui/system";
+import { Typography } from '@mui/material';
+import MuiAppBar from '@mui/material/AppBar';
+import MuiDrawer from '@mui/material/Drawer';
+import { styled } from '@mui/system';
 
 const drawerWidth = 240;
-const mobileDrawerWidth = 140;
+const mobileDrawerWidth = 160;
 
-const openedMixin = (theme) => ({
+const openedMixin = theme => ({
   width: mobileDrawerWidth,
-  [theme.breakpoints.up("sm")]: {
+  [theme.breakpoints.up('sm')]: {
     width: drawerWidth,
   },
-  transition: theme.transitions.create("width", {
+  transition: theme.transitions.create('width', {
     easing: theme.transitions.easing.sharp,
     duration: theme.transitions.duration.enteringScreen,
   }),
-  overflowX: "hidden",
+  overflowX: 'hidden',
 });
 
-const closedMixin = (theme) => ({
-  transition: theme.transitions.create("width", {
+const closedMixin = theme => ({
+  transition: theme.transitions.create('width', {
     easing: theme.transitions.easing.sharp,
     duration: theme.transitions.duration.leavingScreen,
   }),
-  overflowX: "hidden",
+  overflowX: 'hidden',
   width: `calc(${theme.spacing(8)} + 1px)`,
-  [theme.breakpoints.up("sm")]: {
+  [theme.breakpoints.up('sm')]: {
     width: `calc(${theme.spacing(8)} + 1px)`,
   },
 });
 
-const DrawerHeaderStyled = styled("div")(({ theme }) => ({
-  display: "flex",
-  alignItems: "center",
-  justifyContent: "flex-end",
+const DrawerHeaderStyled = styled('div')(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'flex-end',
   padding: theme.spacing(0, 1),
   // necessary for content to be below app bar
   ...theme.mixins.toolbar,
 }));
 
+const AppBarStyled = styled(MuiAppBar, {
```

Comment:
The same here, it was a part of TopMenuStyled.js

Reply:
Yes, I have fixed it now.